routes.go

```diff
@@ -282,11 +280,9 @@ func provideURLPrefix(in provideURLPrefixIn) string {
 }
 
 //nolint:funlen
-func fixV2Duration(getLogger ancla.GetLoggerFunc, config ancla.TTLVConfig, v2Handler http.Handler) (alice.Constructor, error) {
+func fixV2Duration(getLogger sallust.GetLoggerFunc, config ancla.TTLVConfig, v2Handler http.Handler) (alice.Constructor, error) {
 	if getLogger == nil {
```

Comment:
Checking for `nil` here is code smell.  It's ok to require a non-nil logger function to be passed, and simply panic if it isn't

Reply:
yup this is already patched in https://github.com/xmidt-org/tr1d1um/pull/344